models/migrations/v102.go

```diff
@@ -11,5 +11,8 @@ import (
 func dropColumnHeadUserNameOnPullRequest(x *xorm.Engine) error {
 	sess := x.NewSession()
 	defer sess.Close()
-	return dropTableColumns(sess, "pull_request", "head_user_name")
+	if err := dropTableColumns(sess, "pull_request", "head_user_name"); err != nil {
```

Comment:
You have to add `sess.Begin` if you add `sess.Commit`.

Reply:
This was an irrelevant commit - so I've removed it.